solace_exporter.go

```diff
@@ -438,7 +407,7 @@ func (e *Exporter) getSpoolSemp1(ch chan<- prometheus.Metric) (ok float64) {
 	}
 
 	command := "<rpc><show><message-spool></message-spool></show ></rpc>"
-	body, err := postHTTP(e.config.scrapeURI+"/SEMP", e.config.sslVerify, e.config.timeout, "application/xml", command)
```

Comment:
It seam that you removed the no_sll_verification flag. This might be a problem because the semp api if the solace broker normaly use self signed certificates.

Reply:
The flag still exists, but since the `postHTTP()` function is a method on the `Exporter` type now, this variable doesn't need to be passed anymore - it's now simply accessed as `e.config.sslVerify` within the method.